pkgs/applications/radio/js8call/default.nix

```diff
@@ -0,0 +1,62 @@
+{ lib
+, stdenv
+, fetchFromBitbucket
+, wrapQtAppsHook
+, pkg-config
+, hamlib
+, libusb1
+, cmake
+, gfortran
+, fftw
+, fftwFloat
+, qtbase
+, qtmultimedia
+, qtserialport
+}:
+
+stdenv.mkDerivation rec {
+  pname = "js8call";
+  version = "2.2.0";
+
+  src = fetchFromBitbucket {
+    owner = "widefido";
+    repo = pname;
+    rev = "24677a00343c2f3cc53f6762466a4b11893094f8";
+    sha256 = "sha256-4yGxfG1sdOwOOEhoD664TW7rcajUQzRyH/kqTVMpop0=";
+  };
+
+  nativeBuildInputs = [ wrapQtAppsHook ];
+  buildInputs = [
+    pkg-config
```

Comment:
Typically pkg-config goes into nativeBuildInputs since it's executed at build time

Reply:
D'oh, I always mix these up. I have moved some other build time dependencies to nativeBuildInputs, however if I move pkg-config there it fails to find libusb and hamlib. Is there something I need to do to inform pkg-config about these libraries?

It builds fine if I throw everything in to buildInputs within mkShell.